doc/code_style/mom.h

```diff
@@ -113,16 +113,16 @@ class YourMom : public MassiveEntity {
 enum class awesome_enumeration {
 	LOW,
 	MID,
-	HIGH,  // <- note the last , there!
+	HIGH, // <- note the last , there!
 };
 
 
 /**
  * Type aliases are done with "using".
  * They can get proper documentation, too!
  */
-using wtf_t = std::unordered_map<std::string,
-                                 std::unique_ptr<Stuff>>;
+using wtf_t = std::unordered_map<std::string, std::unique_ptr<Stuff>>;
 
-}} // parents::mom
+} // namespace mom
+} // namespace parents
```

Comment:
that's an acceptable change, especially multi-part-namespaces are now allowed with c++17 and require a single {} pair.

Reply:
I need to recheck, but afair it was clang-tidy that merged these together in the files automatically. I will make another run soon when I try out the Git workflows (want to let them export patch files from `clang-format` and `clang-tidy` for the people to make fixing their code easier). So I think that would be something for clang-tidy later, I wanted to say with it.

Reply:
Can confirm, I think what you mean is this one:
https://clang.llvm.org/extra/clang-tidy/checks/modernize-concat-nested-namespaces.html